src/lib/webengine/webview.cpp

```diff
@@ -522,7 +522,7 @@ void WebView::openUrlInNewTab(const QUrl &url, Qz::NewTabPositionFlags position)
 void WebView::openActionUrl()
 {
     if (QAction* action = qobject_cast<QAction*>(sender())) {
-        load(action->data().toUrl());
+        openUrlInNewTab(action->data().toUrl(), Qz::NT_SelectedTab);
```

Comment:
Oh, with the the possibility to open the URL in a new tab with a middle click, this change makes no sense. I think this can be closed (or is there anything else to do in this context?)...